packages/curvenote/src/sync/clone.ts

```diff
@@ -39,26 +38,29 @@ export async function interactiveCloneQuestions(
     }
   }
   let path: string;
-  const defaultPath = join('content', project.data.name);
+  const defaultPath = '.';
   if (opts?.path || opts?.yes) {
     path = opts?.path ?? defaultPath;
     if (path !== '.' && fs.existsSync(path)) {
       throw new Error(`Invalid path for clone: "${path}", it must not exist.`);
     }
   } else {
-    const { projectPath } = await inquirer.prompt([questions.projectPath(defaultPath)]);
-    path = projectPath;
+    path = defaultPath;
```

Comment:
`path` above can still be set as a CLI option, so now it's just no longer an interactive choice.